GOVERNANCE.md

```diff
@@ -0,0 +1,130 @@
+# Release Working Group
+
+The Node.js Release Working Group (WG) maintains oversight
+over the Node.js Release and Long Term Support (LTS) teams. It
+manages the release and long term support shedule and policies
+for all Node.js releases.
+
+The WG has final authority over Releases including:
+
+* Release process and tools.
+* Content for all releases.
```

Comment:
Agreed on dissolving. However if we consider other WGs, this WG will have the responsibility to what *is* Node in the hand of our users, this is very different to most of the other WGs.

> ... the WG would be overstepping its bounds ...

Yes, exactly. I'm just pointing it out that those bounds are not written here. It might not be on this point, but a generic mention to them would be welcomed, maybe at the top.